pipeline.ttl

```diff
@@ -6,27 +6,25 @@
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@base <http://emmo.info/oteio/pipeline> .
+@base <http://emmo.info/oteio/pipeline#> .
 
-<http://emmo.info/oteio/pipeline> rdf:type owl:Ontology ;
-                                           owl:versionIRI <http://emmo.info/0.3.0/oteio/pipeline> ;
+<http://onto-ns.com/meta/oteapi/pipeline> rdf:type owl:Ontology ;
+                                           owl:versionIRI <http://onto-ns.com/meta/oteapi/1/pipeline> ;
```

Comment:
It is standard to place the version part just before the root component that is versioned. With github we are versioning the oteapi repo, not the different ttl files individually. I therefore think that <http://emmo.info/0.3.0/oteio/pipeline> is more standard.